test/integration/client/exec_test.go

```diff
@@ -464,3 +568,61 @@ func loadX509KeyPair(certFile, keyFile string) *tls.Certificate {
 	}
 	return &cert
 }
+
+func createNamespace(ctx context.Context, t *testing.T, client clientset.Interface) *corev1.Namespace {
+	t.Helper()
+
+	ns, err := client.CoreV1().Namespaces().Create(
+		ctx,
+		&corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: "test-exec-plugin-with-informer-ns"}},
+		metav1.CreateOptions{},
+	)
+	if err != nil {
+		t.Fatal(err)
+	}
+	t.Cleanup(func() {
+		// Use a new context since the one passed to this function would have timed out.
+		ctx, cancel := context.WithTimeout(context.Background(), time.Second*10)
+		defer cancel()
+		if err := client.CoreV1().Namespaces().Delete(ctx, ns.Name, metav1.DeleteOptions{}); err != nil {
+			t.Error(err)
+		}
+	})
+
+	return ns
+}
+
+func startConfigMapInformer(ctx context.Context, t *testing.T, client clientset.Interface, namespace string) *informerSpy {
+	t.Helper()
+
+	var informerSpy informerSpy
+	informers := informers.NewSharedInformerFactoryWithOptions(client, 0, informers.WithNamespace(namespace))
```

Comment:
nit: rename `informers` to not conflict with the import name.

Reply:
9a64b4195ab